helm/cluster-eks/values.schema.json

```diff
@@ -396,6 +396,31 @@
                             ],
                             "default": "public"
                         },
+                        "encryptionConfig": {
+                            "type": "object",
+                            "title": "Encryption config",
+                            "description": "Encryption configuration for the Kubernetes API server.",
+                            "properties": {
+                                "keyARN": {
+                                    "type": "string",
+                                    "title": "Key ARN",
+                                    "description": "AWS KMS key ARN. To enable encryption when creating a cluster you need to create a new KMS key that has an alias name starting with cluster-api-provider-aws-. For example, arn:aws:kms:eu-north-1:12345678901:alias/cluster-api-provider-aws-key1.",
+                                    "examples": [
+                                        "arn:aws:kms:eu-west-1:123456789012:key/1234abcd-12ab-34cd-56ef-1234567890ab"
```

Comment:
This differs from the above example (`alias/...` or not?)

Reply:
alias ARN and key ARN are different things,  and they both have their role, the above is corect

Reply:
Okay, but the description should differentiate these. Right now, the description ends with an example for the alias ARN which could be mistakenly filled into `keyArn` by a user. We should even make such mistakes impossible by checking the value in the Helm template.